batches/vep.py

```diff
@@ -0,0 +1,40 @@
+"""
+Batch script that runs a VEP job on input VCFs.
+"""
+import os.path
+from typing import List
+
+from cpg_pipes.hb.batch import setup_batch
+from cpg_pipes.jobs import vep
+import click
+
+
+@click.command()
+@click.option(
+    '--vcf',
+    '--vcf-path',
+    'vcf_paths',
+    multiple=True,
+    help='Paths to VCFs (can be multiple)',
+)
+@click.option(
+    '--project',
+    'project',
+    help='Billing project for Batch'
```

Comment:
Don't we typically set that through an environment variable, e.g. in the analysis-runner?

Reply:
It's an optional parameter, and defaults to the env variable if it's defined:
```
billing_project = billing_project or os.getenv('HAIL_BILLING_PROJECT')
```
Removed the parameter now, reading the env var directly!

Just wanted to avoid it depending on our infra if we want the code to be reused outside of CPG, but could think about it down the road.